_posts/2018-08-06-bashing-through-minikube.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title:  "Bashing through Minikube - 0 to 60 for Local Kubernetes Development"
+date:   2018-08-06
```

Comment:
maybe set the date for Wednesday ie. 2018-08-08

Reply:
I think next Monday would be good - means there's a good flow of posts, but not all at once.